website/docs/r/application_epg.html.markdown

```diff
@@ -64,7 +64,14 @@ resource "aci_application_epg" "fooapplication_epg" {
 
 * `relation_fv_rs_sec_inherited` - (Optional) Relation represents that the EPG is inheriting security configuration from other EPGs (Point to class fvEPg). Cardinality - N_TO_M. Type - Set of String.
 
-* `relation_fv_rs_node_att` - (Optional) Relation used to define a Static Leaf binding (Point to class fabricNode). Cardinality - N_TO_M. Type - Set of String.
+* `relation_fv_rs_node_att` - (Optional) A block representing the relation to a Static Leaf binding (Point to class fabricNode). Cardinality - N_TO_M. Type: Block.
+
+  - `node_dn` - (Required) The Distinguished Name of the Node object. Type: String.
+  - `encap` - (Required) The port encapsulation of the Node Object. Type: String.
+  - `description` - (Optional) The description for relation_fv_rs_node_att of the Node Object. Type: String.
+  - `deployment_immediacy` - (Optional) The deployment immediacy of the Static Path of the Node Object. Allowed values: "immediate", "lazy". Default value: "lazy". Type: String.
+  - `mode` - (Optional) The Application EPG mode of the Node Object. Allowed values: "regular", "native", "untagged". Default value: "regular". Type: String.
+  - `primary_encap` - (Optional) The primary encap for Application EPG of the Node Object. Default value: "unkown". Type: String.
 <!-- tenant -> Application Profile -> EPG ->Static Leaf -->
 
 * `relation_fv_rs_dpp_pol` - (Optional) Relation to define a Data Plane Policing policy (Point to class qosDppPol). Cardinality - N_TO_ONE. Type - String.
```

Comment:
can you add in the "Importing" section the new import method. See this pr for example:
https://github.com/CiscoDevNet/terraform-provider-aci/pull/1084/files